<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QaBoxVM02" failures="2" tests="2" name="CucumberOptions.FailedTestNGRunner" time="30.346" errors="0" timestamp="2025-03-18T11:44:15 UTC" skipped="0">
  <testcase classname="CucumberOptions.FailedTestNGRunner" name="runScenario" time="14.630">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at StepDefination.CheckoutPageSD.User_proceed_to_checkout_and_verify_the_name_in_page(CheckoutPageSD.java:33)
at ✽.User proceed to the checkout page and verify the Beet in the page(file:///C:/Users/Nayan/Documents/Eclipse-Workspace/CucumberActualFramework/src/test/java/feature/Checkout.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="CucumberOptions.FailedTestNGRunner" name="runScenario" time="15.716">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at StepDefination.CheckoutPageSD.User_proceed_to_checkout_and_verify_the_name_in_page(CheckoutPageSD.java:33)
at ✽.User proceed to the checkout page and verify the Tom in the page(file:///C:/Users/Nayan/Documents/Eclipse-Workspace/CucumberActualFramework/src/test/java/feature/Checkout.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- CucumberOptions.FailedTestNGRunner -->
